SwiftCharts/Axis/ChartAxisLayer.swift

```diff
@@ -8,20 +8,48 @@
 
 import UIKit
 
+/// A protocol for all axis layers
 public protocol ChartAxisLayer: ChartLayer {
-        
+
+    /// The location of the smallest axis value
     var p1: CGPoint {get}
+
+    /// The location of the largest axis value
     var p2: CGPoint {get}
+
+    /// The axis values shown in the layer
     var axisValues: [ChartAxisValue] {get}
+
+    /// The bounding rectangle of the layer
     var rect: CGRect {get}
+
+    /// The screen locations that corresponds with the axis values. The screen location for a value is the point coordinate that corresponds with the axis' dimension. TODO improve this wording or provide an example
```

Comment:
I think "The screen locations that corresponds with the axis values" is understandable enough.